frontend/src/todos/todo.tsx

```diff
@@ -1,7 +1,7 @@
 export type Todo = {
   id: string;
   name: string;
```

Comment:
It actually make sense to hold it as a date, you can map the result returned from the back